imap/append.c

```diff
@@ -94,6 +94,8 @@ struct stagemsg {
     struct message_guid guid;
 };
 
+uint64_t append_counter;
```

Comment:
This wants to be `static`.  Among other things, that will guarantee it's initialised to zero when left uninitialised like this...

Reply:
Yep, done.